lib/block_view.go

```diff
@@ -113,6 +114,21 @@ type UtxoView struct {
 	// Map of DeSoNonce and PKID to TransactorNonceEntry
 	TransactorNonceMapKeyToTransactorNonceEntry map[TransactorNonceMapKey]*TransactorNonceEntry
 
+	// Validator mappings
+	ValidatorMapKeyToValidatorEntry map[ValidatorMapKey]*ValidatorEntry
+
+	// Global stake across validators
+	GlobalStakeAmountNanos *uint256.Int
+
+	// Stake mappings
+	StakeMapKeyToStakeEntry map[StakeMapKey]*StakeEntry
+
+	// Locked stake mappings
```

Comment:
Add a comment here and/or in db_utils. The concept of locked stake is a bit tricky.